repository/targets.json

```diff
@@ -1 +1 @@
d64"}},"fioctl-darwin-arm64_v0.34":{"length":20841298,"hashes":{"sha512":"ce8e891ae93923c2421db0af9e39879d6138a74fbbba294be70451c1c41f970e730179c77614ba890c71f817fc1ad73c5bbe341bf51b57ad924383e3252b91c7"},"custom":{"platform":"darwin/arm64","version":"v0.34","uri":"https://github.com/foundriesio/fioctl/releases/download/v0.34/fioctl-darwin-arm64"}},"fioctl-darwin-arm64_v0.37":{"length":24501938,"hashes":{"sha512":"cd998044faf806bc6aa5ae28a6187a23f035d5cebf17a484a7ade37171d646913aa3c80080df7d8df8c8587dc32cfb0a951c54ceccbcf681f4a1b42be511bb62"},"custom":{"platform":"darwin/arm64","version":"v0.37","uri":"https://github.com/foundriesio/fioctl/releases/download/v0.37/fioctl-darwin-arm64"}},"fioctl-linux-amd64_v0.34":{"length":21459194,"hashes":{"sha512":"5d5303a126d828bbc0ae3dfabff26bd9e73cdecbcc6c64efcdc45439542146e3213190369f85c77205cd6dc823578853bd5718c227cf0c29fc1178590729bc20"},"custom":{"platform":"linux/amd64","version":"v0.34","uri":"https://github.com/foundriesio/fioctl/releases/download/v0.34/fioctl-linux-amd64"}},"fioctl-linux-amd64_v0.37":{"length":25238456,"hashes":{"sha512":"e7d9459c6b38587266953d0b217717327bf88bc9a8c82253b4b87f5691281b3a63c0c773d0d4fd9a31912e74d5ab544806ada0978dbded723d341f2e60cb152b"},"custom":{"platform":"linux/amd64","version":"v0.37","uri":"https://github.com/foundriesio/fioctl/releases/download/v0.37/fioctl-linux-amd64"}},"fioctl-linux-arm64_v0.34":{"length":20441538,"hashes":{"sha512":"d1bbb0c871d98ddf1295b94c7ec2d30ba633cdef69fa363f0774f73abde508e47eab7d0c5b99cf49e4e93cbdd7ed1a36f50cdbb8afba6b51834041931b9b7331"},"custom":{"platform":"linux/arm64","version":"v0.34","uri":"https://github.com/foundriesio/fioctl/releases/download/v0.34/fioctl-linux-arm64"}},"fioctl-linux-arm64_v0.37":{"length":24090221,"hashes":{"sha512":"c64b6c24d7f66bc3d6387936f846a508b31862ce18526024191eb823c57d0bdb237c3bcafc0312b938d136242066e934cb8dada92f8a94379a7c97f1a219b07e"},"custom":{"platform":"linux/arm64","version":"v0.37","uri":"https://github.com/foundriesio/fioctl/releases/download/v0.37/fioctl-linux-arm64"}},"fioctl-windows-amd64.exe_v0.34":{"length":22006272,"hashes":{"sha512":"a154ea9bcf69935fe24ad046b86a98aad459a86fff2a57dd4b9b259c7148146faa2a01538a2ed7e4f80ee6f490e198de3434e8b08f30e862ca540ba0b01f4fd2"},"custom":{"platform":"windows/amd64","version":"v0.34","uri":"https://github.com/foundriesio/fioctl/releases/download/v0.34/fioctl-windows-amd64.exe"}},"fioctl-windows-amd64.exe_v0.37":{"length":25697280,"hashes":{"sha512":"159227199ec688d7bff7f695596860338ee6636255d41212494bcc5fc3aa655c050a712b85a7c51870bc9bf25e212ee2967030d75f10475b2a4712b010a4c998"},"custom":{"platform":"windows/amd64","version":"v0.37","uri":"https://github.com/foundriesio/fioctl/releases/download/v0.37/fioctl-windows-amd64.exe"}}}},"signatures":[{"keyid":"5f97b9e6d88054fa8f87636e821ef9a28f6b6dab8bee8c9334e812b704dfd88c","sig":"e9e03c1c4e6b0ea05772fef3490b80b780017e28422322cbd08f8497e76b7d0e7e4bb1e7206cb2453d79649afce04d77f44ca891070dcdf0e4f574dd97d40606"}]}
```

Comment:
it looks like you are removing v35 and v36 and keeping v34?

Reply:
Strange; this is what the script did.

Let me clean and redo.

Reply:
That's probably because I tried to run that script before `git pull`, so there was some garbage in `staged`.
You think we need to add this to the readme?

Now it all looks good.

Reply:
It worked with local fioctl build pointing at this branch:
```
v0.36
Update available: 0.37.0
	 https://github.com/foundriesio/fioctl/releases/download/v0.37/fioctl-linux-amd64

You can update by running:
	 /home/vkhoroz/dev/fioctl/bin/fioctl-linux-amd64 version --update-to 0.37.0
```